bin/deploy

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
```

Comment:
I've seen lately a lot of GitHub actions are built with typescript; it might be good to eventually migrate to that b/c it could be easier to maintain than shell scripts in the long run. That said, we already had some work to leap frog off of with the shell scripts and perhaps this repo won't see too many changes 🤷‍♂️